views/partials/_mobile-navigation.njk

```diff
@@ -0,0 +1,82 @@
+{% from "breadcrumb/macro.njk" import govukBreadcrumb %}
+
+<nav id="app-c-mobile-nav" class="app-c-mobile-nav app-c-mobile-nav--hide-on-pageload js-app-mobile-nav" role="navigation">
+  <ul class="app-c-mobile-nav__list">
+    <li>
+      <a href="/about" class="js-mobile-nav-subnav-toggler">About</a>
```

Comment:
two things:
to keep css classes consistent: `app-c-mobile-nav__subnav-toggler` ?
should we attach js to the wrapping container and then do ` .js app-c-mobile-nav__subnav-toggler`